### PR TITLE
Updata doc

### PR DESCRIPTION
**Pull request by freeliuzc**
### PR types

Others
### PR changes

Docs
### Describe

Update docker chapter in installation_guide.

### CODE REVIEW

File: docs/source/install_cn.md [hunk 1]
```diff
@@ -131,25 +131,13 @@ pip install . -i https://pypi.tuna.tsinghua.edu.cn/simple
...
-```bash
-cd /PaddleSpeech
-```
+Docker 是一种开源工具，用于在和系统本身环境相隔离的环境中构建、发布和运行各类应用程序。如果您没有 Docker 运行环境，请参考 [Docker 官网](https://www.docker.com/)进行安装，如果您准备使用 GPU 版本镜像，还需要提前安装好 [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) 。 
+
+PaddleSpeech 提供了带有最新代码的 docker 镜像供您使用，您只需要**拉取 docker 镜像 **，然后**运行 docker 镜像**，无需其他任何额外操作，即可开始使用 PaddleSpeech 的所有功能。
+
+在 [Docker Hub](https://hub.docker.com/repository/docker/paddlecloud/paddlespeech) 中获取这些镜像及相应的使用指南，包括 CPU、GPU、ROCm 版本。
+
+如果您对自动化制作docker镜像感兴趣，或有自定义需求，请访问 [PaddlePaddle/PaddleCloud](https://github.com/PaddlePaddle/PaddleCloud/tree/main/tekton) 做进一步了解。
```
Comment (yt605155624) [inline docs/source/install_cn.md:140]: docker 英文前后请用空格隔开

**Review by Jackwaterveg — DISMISSED** [review]
LGTM

File: docs/source/install_cn.md (hunk 1 above)
Comment (freeliuzc) [reply docs/source/install_cn.md:140]: 已更新

**Review by yt605155624 — APPROVED** [review]
LGTM

**Pull request merged**